Bugzilla/Mailer.pm

```diff
@@ -154,8 +154,10 @@ sub MessageToMTA {
       WARN("Attempted to send email to address in badhosts: $to");
       $email->header_set(to => '');
     }
-    elsif ($recipient->host =~ /\.(?:bugs|tld)$/) {
-      WARN("Attempted to send email to fake address: $to");
+    elsif ($recipient->host =~ /\.(?:bugs|tld)$/
+      || $recipient->address eq 'nobody@mozilla.org')
```

Comment:
Can you put this logic into a separate function and then use it from both locations? Maybe `is_fake_email(email_address)` or something like that.